build.sbt

```
@@ -155,9 +153,7 @@ lazy val testsOutput = project
                          Seq("-P:semanticdb:synthetics:on")
                        else Nil),
     scalacOptions -= warnUnusedImports.value,
-    libraryDependencies ++= (if (scalaVersion.value.startsWith("2"))
-                               testsDeps :+ "org.scala-lang" % "scala-reflect" % scalaVersion.value
-                             else Nil),
+    libraryDependencies ++= testDependencies.value,
```

this is about the tests module, not the test configuration, so I think plural matters
```suggestion
    libraryDependencies ++= testsDependencies.value,
```